vhdl/vhdl.g4

```diff
@@ -956,32 +956,37 @@ multiplying_operator
 //     ;
 // changed to avoid left-recursion to name (from selected_name, indexed_name,
 // slice_name, and attribute_name, respectively)
-// (2.2.2004, e.f.)
+// (2.2.2004, e.f.) + (12.07.2017, o.p.)
 name
-  : selected_name  
-  | name_part ( DOT name_part)*
+  : ( identifier | STRING_LITERAL ) ( name_part )*

```

Comment:
Do not use whitespaces after and before open and close parenthesis respectively.